docs/index.md

```diff
@@ -1 +1,7 @@
 # Welcome to the Vignette wiki!
+
+This is the place that stores the cummulative knowledge of the vignette community. To get started with Vignette, visit [The Windows Installation Guide](install/install-win.md) or [The Linux Installation Guide](install/install-linux.md).
```

Comment:
Fixed grammar, capitalization and pointed the user towards the sidebar for all other pages.

```suggestion
This is the place that stores the cumulative knowledge of the Vignette community. 

To get started with Vignette, you can go to [The Windows Installation Guide](install/install-win.md) or [The Linux Installation Guide](install/install-linux.md).

Or, if you are looking for a specific page, you can use the sidebar instead.
```